src/packet.proto

```diff
@@ -2,6 +2,23 @@ syntax = "proto3";
 
 package helium;
 
+message gps_info {
+    uint32 horizontal_accuracy = 1; // meters? this will take into account h3 precision and distance from asserted location
+    uint32 vertical_accuracy = 2; // meters?
+    int32 altitude = 3; // meters?
+    uint32 time_accuracy = 4; //nanoseconds
```

Comment:
Should this be uint64?

Reply:
it is typically in the range of 20-200ns but maybe?

Reply:
It's sourced from UBX NAV-PVT which represents it as a u32

```tAcc ns Time accuracy estimate (UTC)```